app/templates/index.hbs

```diff
@@ -44,7 +46,11 @@
                 <div class="col-xs-12">
                     <h2>{{t "index.subjects.heading"}}</h2>
                     <div class="subjectsList p-lg"> {{!SUBJECTS LIST}}
-                        {{taxonomy-top-list list=model.taxonomies}}
+                        {{#if theme.provider.additionalProviders}}
+                            {{additional-provider-list additionalProviders=theme.provider.additionalProviders}}
```

Comment:
The intention was to have additional providers be separate from allowing submissions but allowing submissions to be tied to not having taxonomies. That being said, yes, getting this out the door is key, but we should make follow-up tickets to clean this up immediately so that we don't have to maintain the hacks long-term and we can actually do it properly.